ocs_ci/ocs/pgsql.py

```diff
@@ -258,6 +278,68 @@ def validate_pgbench_run(self, pgbench_pods):
             all_pgbench_pods_output.append(pg_output)
         return all_pgbench_pods_output
 
+    def get_nodes(self, pod_name, all_nodes=False):
+        """
+         Get nodes that contain a specific pod
+
+        Args:
+            pod_name (str)
+            all_nodes (bool):
+                True-return all the nodes that contain postgres pod
+                False-return 1 node that contain postgres pod
+        Returns:
+            list: Cluster node OCP objects
+
+        """
+        if pod_name == 'osd':
+            pods_obj = self.pod_obj.get(selector=constants.OSD_APP_LABEL, all_namespaces=True)
+        elif pod_name == 'postgres':
+            pods_obj = self.pod_obj.get(selector=constants.PGSQL_APP_LABEL, all_namespaces=True)
+        # Create a list of nodes (witout duplicate nodes in the list)
+        nodes_set = list(set([pod['spec']['nodeName'] for pod in pods_obj['items']]))
+
+        if all_nodes:
+            node_list = get_node_objs(nodes_set)
+            return node_list
+
+        # Selects one Node (random)
+        node_obj = get_node_objs(nodes_set[random.randint(0, len(nodes_set) - 1)])
+        return node_obj
+
+    def respin_pod(self, pod_name=''):
+        """
+        Respin a pod
+
+        Args:
+            pod_name (str)
+
+        Returns:
+            pod status
+        """
+        log.info(f"Respin pod {pod_name}")
+        if pod_name == 'postgers':
+            self.respin_app_pod()
+            return True
+        disruption = disruption_helpers.Disruptions()
+        disruption.set_resource(resource=f'{pod_name}')
+        disruption.delete_resource()
+        return True
+
+    def respin_app_pod(self):
+        """
+        Respin the pgsql app pod
+
+        Returns:
+            pod status
+        """
+        app_pod_list = get_operator_pods(constants.PGSQL_APP_LABEL, constants.RIPSAW_NAMESPACE)
+        app_pod = app_pod_list[random.randint(0, len(app_pod_list) - 1)]
+        app_pod.delete(wait=True, force=False)
+        wait_for_resource_state(
+            resource=app_pod, state=constants.STATUS_RUNNING, timeout=300
+        )
+        return True
```

Comment:
if a function returns bool it needs to return either True or False. Here there is an option to return only True